src/OrbitVulkanLayer/VulkanLayerController.h

```diff
@@ -704,15 +704,14 @@ class VulkanLayerController {
     }
   }
 
-  void AddRequiredDeviceExtensionNameIfMissing(
-      const VkDeviceCreateInfo* create_info, VkPhysicalDevice physical_device,
-      PFN_vkGetInstanceProcAddr next_get_instance_proc_addr_function, const char* extension_name,
-      std::vector<std::string>* output) {
+  void AddRequiredDeviceExtensionNameIfMissing(const VkDeviceCreateInfo* create_info,
+                                               VkPhysicalDevice physical_device,
+                                               const char* extension_name,
+                                               std::vector<std::string>* output) {
     auto raw_enumerate_device_extension_properties_function =
         // Pass a valid instance, as following the spec. we are not allowed to use nullptr here.
```

Comment:
Is this still valid?

Reply:
I guess this is the only open comment for me.

Reply:
It is still valid, but, I would argue that it now confuses more than it adds, so I am going to remove it. It is pretty clear what is going on here, anyways.